src/DynamoCore/DynamoCore.csproj

```diff
@@ -315,11 +325,13 @@ IF NOT EXIST "$(OutputPath)\dll" "$(OutputPath)\InstallASMForDynamo.exe" /SILENT
 copy /Y "$(SolutionDir)..\extern\LibG\x64\Release\*" "$(OutputPath)\dll"
 </PostBuildEvent>
     <PostBuildEvent>xcopy /Y "$(SolutionDir)..\extern\ProtoGeometry\*" "$(OutputPath)"
-xcopy  /Y "$(SolutionDir)..\doc\distrib\License.rtf" "$(OutputPath)"
+xcopy /Y "$(SolutionDir)..\extern\ProtoGeometry\ProtoGeometry.xml" "$(OutputPath)\$(UICulture)"
```

Comment:
The thing is, `ProtoGeometry.xml` is generated when `ProtoGeometry.dll` is built on a separate repo, we just have this step here to copy the XML documentation to `$(OutputPath)`. That's the existing behaviour, during localization that needs to be copied to `$(OutputPath)\$(UICulture)` instead.

Also, `License.rtf` receives the same treatment.